definitions/npm/react-dropzone_v7.x.x/test_react-dropzone_v7.x.x.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
```

Comment:
Test file should be placed in the flow-versioned folder, alongside the implementation.